sys/net/gnrc/Makefile.dep

```diff
@@ -125,6 +125,7 @@ endif
 ifneq (,$(filter gnrc_ipv6_auto_subnets,$(USEMODULE)))
   USEMODULE += gnrc_ipv6_nib_rtr_adv_pio_cb
   CFLAGS += -DCONFIG_GNRC_IPV6_NIB_ADV_ROUTER=0
+  CFLAGS += -DCONFIG_GNRC_IPV6_NIB_RTR_ADV_ON_RA=0
```

Comment:
If the 6LO interface is the *downstream* interface, we must be a border router, so this would be `0` anyway.
If the 6LO interface is the *upstream* interface, we must set this option to ensure correct behavior.